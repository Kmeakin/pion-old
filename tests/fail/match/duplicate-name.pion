//~ run = "${PION} elab ${FILE}"

{
let (x, y, x) = (true, false, true);
{}
}
