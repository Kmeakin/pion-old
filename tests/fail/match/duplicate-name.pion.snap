exit status: 0
stdout:
let x : Bool = (true, false, true)._0;
let y : Bool = (true, false, true)._1;
() : ()

stderr:
error: duplicate definition of local name `x`
  ┌─ tests/fail/match/duplicate-name.pion:4:12
  │
4 │ let (x, y, x) = (true, false, true);
  │      -     ^ duplicate definition
  │      │      
  │      first definition

